### PR TITLE
Kempe (Linear) Algorithm implemented in Java

### PR DESCRIPTION
**Pull request by Riyazul555**
Hey @iuliagroza check this out......pR for #44.....
Do tell me if nay changes are required
Thanks

### CODE REVIEW

File: Kempe/Kempe(Linear).java [hunk 1]
```diff
@@ -0,0 +1,125 @@
+//This code is contributed by Riyazul555
+
+// Time Complexity = O(N^2 + N * (N+M))
```
Comment (iuliagroza) [inline Kempe/Kempe(Linear).java:3]: We're working on dense graphs, so this can be simplified to O(N*(N+M)), since N*(N+M) > N² anyway.

Comment (iuliagroza) [inline Kempe/Kempe(Linear).java:3]: Can you please also specify what N and M denote (see [kruskal.cpp](https://github.com/iuliagroza/Algorithms/blob/2de20024f23e3fcfd915c3474a0b49c291f4e173/Kruskal/kruskal.cpp#L3))?

**Comment by Riyazul555** [comment]
Ya sure @iuliagroza .....I will fix that....By the way thanks for your correction.

**Comment by Riyazul555** [comment]
Hey @iuliagroza  I have fixed the changes.....please review it once

**Review by iuliagroza — APPROVED** [review]

**Pull request merged**